bundles/best_practices.rst

```diff
@@ -68,30 +68,41 @@ The basic directory structure of an AcmeBlogBundle must read as follows:
 .. code-block:: text
 
     <your-bundle>/
-    ├─ AcmeBlogBundle.php
-    ├─ Controller/
-    ├─ README.md
-    ├─ LICENSE
-    ├─ Resources/
-    │   ├─ config/
-    │   ├─ doc/
-    │   │  └─ index.rst
-    │   ├─ translations/
-    │   ├─ views/
-    │   └─ public/
-    └─ Tests/
+    ├── config/
+    ├── docs/
+    │   └─ index.rst
+    ├── public/
+    ├── src/
+    │   ├── Controller/
+    │   ├── DependencyInjection/
+    │   └── AcmeBlogBundle.php
+    ├── templates/
+    ├── tests/
+    ├── translations/
+    ├── LICENSE
+    └── README.md
+
+and the bundle path must be adjusted to the root directory::
+
+    class AcmeBlogBundle extends Bundle
+    {
+        public function getPath(): string
+        {
+            return \dirname(__DIR__);
+        }
+    }
 
 **The following files are mandatory**, because they ensure a structure convention
 that automated tools can rely on:
 
-* ``AcmeBlogBundle.php``: This is the class that transforms a plain directory
+* ``src/AcmeBlogBundle.php``: This is the class that transforms a plain directory
   into a Symfony bundle (change this to your bundle's name);
 * ``README.md``: This file contains the basic description of the bundle and it
   usually shows some basic examples and links to its full documentation (it
   can use any of the markup formats supported by GitHub, such as ``README.rst``);
 * ``LICENSE``: The full contents of the license used by the code. Most third-party
   bundles are published under the MIT license, but you can `choose any license`_;
-* ``Resources/doc/index.rst``: The root file for the Bundle documentation.
+* ``docs/index.rst``: The root file for the Bundle documentation.
```

Comment:
Maybe we should not enforce the usage of `.rst` as the documentation format. AFAICT, much more bundles are using `.md` because they rely on github to render the doc anyway, not on an external website built with Sphinx (and Github's `rst` rendering does not support all Sphinx features anyway).

Reply:
btw, later, we allow changing the format